crates/mpz-core/src/lpn.rs

```diff
@@ -64,7 +64,7 @@ impl<const D: usize> LpnEncoder<D> {
 
     #[inline]
     fn compute_one_row(&self, y: &mut [Block], x: &[Block], pos: usize, prp: &Prp) {
-        let block_size = (D + 4 - 1) / 4;
+        let block_size = D.div_ceil(4);
```

Comment:
I am not sure if it is helpful here to follow clippy's advice. I suspect that this can create confusion so I would tend to suppress clippy or add a comment.

Reply:
I assume the Clippy team made this rule with severity `error` for a reason.
The docs says: **It's simpler, clearer and more readable.**

https://rust-lang.github.io/rust-clippy/master/index.html#manual_div_ceil
https://github.com/rust-lang/rust-clippy/issues/12894

Reply:
I don't have a strong opinion here. My thoughts were that I would probably spend a few extra brain cycles what happened to the original formula.